server/src/lib/api.ts

```diff
@@ -116,13 +117,39 @@ export default class API {
 
     router.use('/challenge', this.challenge.getRouter());
 
+    router.get('/feature/:locale/:feature', this.getFeatureFlag);
+
     router.use('*', (request: Request, response: Response) => {
       response.sendStatus(404);
     });
 
     return router;
   }
 
+  getFeatureFlag = (
+    { params: { locale, feature } }: Request,
+    response: Response
+  ) => {
+    let featureResult = null;
+
+    try {
+      const featureToken = feature as FeatureToken;
+      if (featureTokens.includes(featureToken)) {
+        const featureTemp = features[featureToken];
+        if (
+          (!featureTemp.locales || featureTemp.locales.includes(locale)) &&
+          getConfig()[featureTemp.configFlag as keyof CommonVoiceConfig]
```

Comment:
This is ugly, but without the `as keyof` typescript throws a bunch of `implicit any` errors, and I wanted to avoid modifying `CommonVoiceConfig` itself as much as possible since it's used in so many places. If you know of a different way of accessing a variable key, please do let me know!

Reply:
It does if I create another type called `FeatureFlag` or something and directly specify `BENCHMARK_LIVE` as the string literal, but that also felt ugly because there would be a set of string literals for `FeatureToken`, another corresponding set for `FeatureFlag`, and then the `FeatureType` that combines the two sets in a way where if you want to make changes you'd have to update several places. 

Instead of specifying `BENCHMARK_LIVE` in `CommonVoiceConfig`, I did consider something like:

```
FEATURE_FLAGS: {
   [key: string]: boolean;
}
```

But then the problem was to cleanly set default values for an unknown set of potential env vars. None of these options are particularly great.